com.unity.render-pipelines.universal/Runtime/Passes/CopyColorPass.cs

```diff
@@ -143,21 +143,21 @@ public override void Execute(ScriptableRenderContext context, ref RenderingData
                 ScriptableRenderer.SetRenderTarget(cmd, destination, k_CameraTarget, clearFlag,
                     clearColor);
 
-                bool useDrawProceduleBlit = renderingData.cameraData.xr.enabled;
+                bool useDrawProceduleBlit = SystemInfo.graphicsShaderLevel >= 30;
```

Comment:
Remove this

Reply:
Yes, if we're sure no GLES or WebGL1 is required anymore.